Examples/Python/cinematic_rendering/head_and_neck_animation.py

```diff
@@ -0,0 +1,206 @@
+"""Generate a camera animation of the head and neck example
+
+Running this script generates a video quilt file that can be
+played on a Looking Glass device via HoloPlay Studio. While the
+script is running, the latest frame generated can be seen on the
+device. However, due to computing constraints, the time between the
+frames may be significantly longer than that of the video file that
+is produced.
+
+Note that HoloPlay Studio requires the video file to be in MP4
+or WEBM format. If VTK writes out the video in a different format,
+it may need to be converted. Here's an example of a conversion
+using ffmpeg:
+
+ffmpeg -i input_file.ogv -c:v libx265 -crf 10 output_file.mp4
+
+Where `libx265` specifies to use the h.265 video codec, and `crf`
+specifies the video quality (where 0 is lossless and 51 is the lowest
+quality).
+
+Example output of this script that was generated for a 16" device and
+converted to MP4 may be found here:
+
+https://drive.google.com/file/d/1WUFDhrcLsajCV7TUoNKFCII1VGNHcrz_/view?usp=sharing
```

Comment:
Is this accessible without a google account?

Reply:
Yes it is!